security/passage/Portfile

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8::et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        FiloSottile passage 1.7.4a1
+categories          security
+license             GPL-2+
+platforms           any
+supported_archs     noarch
+
+name                passage
```

Comment:
"name" and "version" fields are set from the "github.setup" line so please remove